bip-0341.mediawiki

```diff
@@ -294,7 +294,17 @@ Examples of preimage for sighashing for each of the sighash modes.
 
 == Deployment ==
 
-TODO
+This BIP is deployed concurrently with [[bip-0342.mediawiki|BIP342]].
+
+For Bitcoin signet, these BIPs are always active.
+
+There may be multiple deployment attempts of these BIPs.
+
+The first deployment will use BIP 8 with the name "taproot", using bit 2, and a lockinontimeout of false.
+
+For Bitcoin mainnet, the BIP 8 startheight will be 681408 (approximately May 1st 2021), the timeoutheight will be 695520 (approximately August 6th 2021), the minimum_activation_height will be 709632 (approximately November 12th 2021), and the activation threshold will be 1815 out of 2016 blocks (90%).
+
+For Bitcoin testnet, TBD.
```

Comment:
Maybe elaborate here:

> For Bitcoin testnet, TBD. Due to the difficulty reset, the pace of testnet block generation is unpredictable. This makes height based activation impractical.

Reply:
Or alternatively:

"For Bitcoin testnet, Taproot will be activated if and when Taproot is active on mainnet. Taproot is already active on signet for testing and experimentation purposes."

Reply:
Added these proposed heights.